pkg/config/pipeline.go

```diff
@@ -15,7 +15,7 @@ import (
 
 var (
 	DefaultEmailTemplate = template.Must(template.New("email").Parse(`
-A file has been {{ .Verb }}ed from {{ .CompanyName }}: {{ .Filename }}
+A file has been {{ .Verb }}ed {{ if and (eq .Verb "upload") (.Hostname) }}to {{ .Hostname }} {{ end }}from {{ .CompanyName }}: {{ .Filename }}
```

Comment:
Why only on upload? I think `{{ if (.Hostname) }}` would be good enough. It's helpful to know if files fail to download as well.

Reply:
I can switch it. I did upload only because of the description in the issue:
> When getting an email / slack message **on file upload** it's helpful to know where it was uploaded to. This could be the hostname for now.

Reply:
Ahh okay. I think we should have it on both, so I probably was focused on upload errors at the time.